ci-operator/jobs/openstack-k8s-operators/glance-operator/openstack-k8s-operators-glance-operator-main-presubmits.yaml

```diff
@@ -124,6 +124,67 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )glance-operator-build-deploy-kuttl,?($|\s.*)
+- agent: kubernetes
```

Comment:
@bshewale we should probably have an issue on this file [1]

[1] https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/37104/pull-ci-openshift-release-master-config/1664246351929544704